docs/getting_started/update_cluster.md

```diff
@@ -0,0 +1,45 @@
+# Updating a Cluster
```

Comment:
You'll have to index this in the docs as well https://github.com/m3db/m3db-operator/blob/master/mkdocs.yml#L60